interfaces/builtin/appstream_metadata.go

```diff
@@ -51,8 +51,11 @@ const appstreamMetadataConnectedPlugAppArmor = `
 
 # Allow access to AppStream collection metadata
 /usr/share/app-info/** r,
+/usr/share/swcatalog/** r,
```

Comment:
To indicate these are equivalent / alternate locations, I wonder if it would be more reabable to either place all the `swcatalog` versions together and below all the `app-info` ones along with a comment - OR use the alternates syntax of AppArmor RegEx's:

```
/usr/share/{app-info,swcatalog}/** r,
/var/cache/{app-info,swcatalog}/** r,
/var/lib/{app-info,swcatalog}/** r,
```

Reply:
That does seem more readable.